opentelemetry/src/context.rs

```diff
@@ -78,6 +84,7 @@ thread_local! {
 pub struct Context {
     #[cfg(feature = "trace")]
     pub(super) span: Option<Arc<SynchronizedSpan>>,
+    pub suppress_logs: bool,
```

Comment:
You are experimenting with logs, but the final expectation is that this will be applicable to traces and logs? If so, the name can be suppress_instrumentation

Reply:
Good comment. Got similar suggestion from @shaun-cox. I changed it to `suppression`, but probably `suppress_instrumentation` is better name.